packages/frontend/components/save-post/form-utils.ts

```diff
@@ -48,3 +49,14 @@ export const SellerFields = (t: TFunction): SellerFieldsType => ({
     title: t('create-post:sellerSelect.otherUser'),
   },
 });
+
+export const ConditionFields = (t: TFunction): ConditionFieldsType => ({
+  NEW: {
```

Comment:
It's better to use [ConditionFieldsType.new]

Reply:
![image](https://user-images.githubusercontent.com/71765510/189296631-b2614264-f367-428c-8c8c-1213161e1945.png)

Reply:
yes, because it's not enum, but why? use enum or at least camelCase for properties